daly-bms-uart.h

```diff
@@ -2,6 +2,10 @@
 #define DALY_BMS_UART_H
 
 #define XFER_BUFFER_LENGTH 13
+#define MIN_NUMBER_CELLS 1
+#define MAX_NUMBER_CELLS 48
```

Comment:
The BMS send the value of max cell and temp Sensors, so i have used This Data to give a max Range, when This Data Not avaible from the BMS the following requests are useless, please Check it, i think variables are better than fixed values for this